src/content/whats-new/2021/04/distributed-tracing-mobile.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Distributed tracing for Mobile'
+summary: 'Get visibility to the entire journey of requests, originating in your mobile app as they travel through distributed systems'
+releaseDate: '2021-04-14'
+learnMoreLink: 'https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-mobile-and-dt/'
+---
+
+New Relic mobile monitoring agents now support distributed tracing! This new feature enables you to see your mobile app activity and connects it directly to the related services that power your app’s experience. By having this enabled, you are now able to see your mobile front-end spans and back-end spans in the same waterfall view.
+
+**What do I need to do to take advantage of Distributed Tracing for Mobile?**
+
+Nothing! It has been turned on by default. If you have mobile agents running, you may have already started seeing distributed tracing data. If you aren’t seeing mobile distributed tracing data, you may need to update your agents.
+
+**Will this increase my data volume?**
+
+Yes it will. If you’d like to turn off distributed tracing for mobile, please read these instructions for [Android](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/api-guides/android-agent-configuration-feature-flags/#dt) and [iOS](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags/#dt).
```

Comment:
Dayna suggested we add a sentence like this (I assume at the end of this sentence): Learn more about  iOS compatibility by clicking **Learn more** above.